docs/variants/pc_engines/initial-deployment.md

```diff
@@ -0,0 +1,42 @@
+# Initial deployment
+
+Initial flashing of Dasharo firmware can be done from Linux using flashrom with
+the internal programmer.
+
+## Deploy using Dasharo Tools Suite
+
+For simplicity we recommend using
+[Dasharo Tools Suite](../../dasharo-tools-suite/overview.md) to
+omit all compilation steps and deploy the Dasharo firmware seamlessly.
+
+1. Boot Dasharo Tools Suite.
+2. Perform Dasharo installation.
+
+After the successful operation, DTS will reboot the platform.
+
+This concludes Dasharo deployment process using DTS.
```

Comment:
I would say that in DTS documentation, we already should have this procedure; if that is the case, we should just link it and avoid multiple sources of truth.

Reply:
https://github.com/Dasharo/docs/pull/794/commits/e59c85b042692ab32d4c97c05a3c2367d9eb1534